full-service/src/config.rs

```diff
@@ -97,7 +97,7 @@ impl APIConfig {
                     signature.product_id(),
```

Comment:
Optional/Equivalent: This `MrSignerVerifier::new` call can be replaced with `MrSignerVerifier::from(signature)`

(I can't comment on that line)